releasenotes/notes/1.4/prepare-1.4.0-aee22d10333374c8.yaml

```diff
@@ -0,0 +1,13 @@
+---
+prelude: |
+    The Qiskit v1.4.0 release is the final minor version release for the 1.x series.
+    This release primarily adds new deprecation warnings for API changes coming
+    in the future major version release 2.0.0. It is fully compatible with the
+    Qiskit 1.3.x releases. It is strongly recommended that you upgrade from
+    1.3.x to 1.4.0 so that you're able to see the warnings about which
+    interfaces will change with Qiskit 2.0.0.
+    The 1.4.x release series will continue to be supported and recieve
+    bugfix and security fixes via patch releases for 6 months after this
```

Comment:
I thought we were planning to have 6 months for general bugfixes and 1 yr for security fixes?

Reply:
whoops, forgot about the specific timelines. Thanks for catching that.